From fda420ff45003d5c3ee3f1333c97d875c0b14a00 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Fri, 13 Jan 2023 12:12:46 +0100
Subject: [PATCH 0108/1141] arm64: defconfig: add stm32 rtc support in addons
 fragment

Enable STM32 RTC with fragment-02-defconfig-addons.config

Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: Ib8e1cf72b82297b5ded89a22cff5babc5c643e64
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/303720
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -4,6 +4,7 @@ CONFIG_ARM_SCMI_CPUFREQ=y
 CONFIG_ARM_SMC_WATCHDOG=y
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_REGULATOR_ARM_SCMI=y
+CONFIG_RTC_DRV_STM32=y
 CONFIG_SERIAL_STM32=y
 CONFIG_SERIAL_STM32_CONSOLE=y
 CONFIG_STM32_DMA3=y
